src/System.Console/tests/ReadAndWrite.cs

```diff
@@ -30,6 +30,14 @@ public static void WriteOverloads()
             Console.SetOut(savedStandardOutput);
         }
     }
+    
+    [Fact]
+    [OuterLoop]
+    public static void WriteToOutputStream()
+    {
+        Stream outStream = Console.OpenStandardOutput();
+        outStream.Write(new byte[] { }, 0, 0);
```

Comment:
Seems like it doesn't both need to be outer loop and write 0 bytes, and should be changed to either be inner loop or write some non-empty line.

Also, do we need a similar test for error stream? What about reading from input stream?